src/kernel/src/fs/mod.rs

```diff
@@ -272,6 +273,27 @@ impl Fs {
         Ok(written.into())
     }
 
+    fn sys_writev(self: &Arc<Self>, i: &SysIn) -> Result<SysOut, SysErr> {
+        let fd: i32 = i.args[0].try_into().unwrap();
+        let ptr: *const libc::iovec = i.args[1].into();
```

Comment:
Yeah and furthermore, there is no libc on Windows, this should be our own type. And in general, I think we should use our own types in the kernel and only use things like libc when communicating directly with the host system